mappings/net/minecraft/util/Util.mapping

```diff
@@ -234,7 +240,15 @@ CLASS net/minecraft/unmapped/C_lvarvugc net/minecraft/util/Util
 		METHOD apply apply (Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;
 	CLASS C_qgeblktr
 		METHOD getAsBoolean getAsBoolean ()Z
+	CLASS C_qygfqbqw
+		METHOD onTermination onTermination (Ljava/lang/Throwable;)V
+			ARG 1 throwable
 	CLASS C_tceggguc
 		FIELD f_boyitqoh cache Ljava/util/Map;
 		METHOD apply apply (Ljava/lang/Object;)Ljava/lang/Object;
 	CLASS C_uzjrvfhh IdentityHashStrategy
+		METHOD equals equals (Ljava/lang/Object;Ljava/lang/Object;)Z
+			ARG 1 object
```

Comment:
```suggestion
			ARG 1 object1
```
If the second param is numbered I think the first should be as well.

Also, on [abbreviation convensions](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#abbreviations):
> "o" for the parameter of equals(Ljava/lang/Object;)Z methods

I know this isn't actually overriding that method but it's clearly patterned after it, so it's worth considering naming the params `o1` and `o2` here.

I'm not sure though, other reviewers please comment.

Reply:
I'm not exactly sure about this really, but if another reviewer agrees with this change I'll implement it.